src/language/rule/abstractRule.ts

```diff
@@ -61,17 +60,6 @@ export abstract class AbstractRule implements IRule {
     ): RuleFailure[] {
         const ctx = new WalkContext(sourceFile, this.ruleName, options);
         walkFn(ctx);
-        return this.filterFailures(ctx.failures);
-    }
-
-    protected filterFailures(failures: RuleFailure[]): RuleFailure[] {
```

Comment:
breaking change. keep this method, just return the input unchanged and deprecate it